### PR TITLE
ideas/list: if wrong backend or no ideas fetched

### PR DESCRIPTION
**Pull request by khamui**
If we do not have the right Backend set, it will fail! This shows a message that helps maybe to set it.

### CODE REVIEW

File: containers/Ideas/IdeasList.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+        props.ideas.map(idea => {
+          return (
+            <IdeasListItem key={`pk${idea.pk}`} idea={idea} {...otherProps} />
+          );
+        })
+      ) : (
+        <TextSourceSans>
+          Could not fetch ideas. Do you use the correct Backend and is your
+          local adhocracy+ server running?
+        </TextSourceSans>
```
Comment (phillimorland) [inline containers/Ideas/IdeasList.js:23]: should we also add text for if there are just no ideas like we do on a+ with 'Nothing to show'?

**Comment by fuzzylogic2000** [comment]
What is the status here?

File: containers/Ideas/IdeasList.js [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
 
-import { IdeasListItem } from './IdeasListItem';
+import {IdeasListItem} from './IdeasListItem';
+import {TextSourceSans} from '../../components/TextSourceSans';
 
-export const IdeasList = (props) => {
+export const IdeasList = props => {
   //do not pass ideas to child components
   const {ideas, ...otherProps} = props;
+  const hasIdeas = !!ideas && ideas.length > 0;
```
[the remark this answers is not in the document]
Comment (khamui) [reply containers/Ideas/IdeasList.js:10]: i don't remember why i did it that way, changed it..

File: containers/Ideas/IdeasList.js (hunk 1 above)
Comment (khamui) [reply containers/Ideas/IdeasList.js:23]: hehe, yes i did it more user oriented now.

**Review by phillimorland — APPROVED** [review]
perfect! :)

**Pull request merged**